src/Epinova.JsResourceHandler.nuspec

```diff
@@ -12,7 +12,7 @@
     <copyright>Epinova AS 2016</copyright>
     <tags>Epinova Episerver Javascript Resources</tags>
     <dependencies>
-      <dependency id="EPiServer.CMS.Core" version="[10.0.0, 11.0.0)"/>
+      <dependency id="EPiServer.CMS.Core" version="[10.0.0, 11.4.0)"/>
```

Comment:
This should be " version="[10.0.0, 12.0.0)" "

https://world.episerver.com/blogs/Quan-Mai/Dates/2017/6/episerver-breaking-changes-policies-/